src/EditorFeatures/CSharpTest/CodeActions/ExtractMethod/ExtractMethodTests.cs

```diff
@@ -1982,15 +1982,20 @@ static void Main(string[] args)
     {
```

Comment:
> Add ImportAdder.Annotation by default in codeGenerationService 

I'm glad this was a separate commit.  It likely needs to be discussed by IDE team if htis should happen by default.  I think i'm ok with it (though it's potentially a big change for people sitting on the IDE API).

An alternative approach would be something like `var generator = SyntaxGenerator.ImportAddingGenerator`.

Reply:
Oh... CodeGenerationService, not SyntaxGEnerator.  n/m.  this should be fine